gradle/java/javac.gradle

```diff
@@ -85,3 +85,23 @@ allprojects {
     }
   }
 }
+
```

Comment:
Can you move it to the gradle file dealing with modules, @mocobeta ?

Reply:
Or, you know what - wait. I'll take another look today. I do have an idea how it can be done (I think).

Reply:
Yes. This just shows one possible solution I could implement. If there is a better/cleaner way this should be replaced with it.